caproto/_circuit.py

```diff
@@ -56,7 +56,8 @@ def __init__(self, our_role, address, priority):
         self.address = address
         self.priority = priority
         self.channels = {}  # map cid to Channel
-        self.log = logging.getLogger("[{} circuit]".format(abbrev))
+        logger_name = f"{abbrev} {address[0]}:{address[1]}-{priority}"
```

Comment:
While separating the logging streams is good, this will be super annoying to pick out the one we want to apply filtering.

Reply:
If I add a `.` in here, it will be easy to do either one:

```
#demo.py
import logging

def spam():
    circ1 = logging.getLogger('cli-circuit.1')
    circ2 = logging.getLogger('cli-circuit.2')
    circ1.info('hello from 1')
    circ2.info('hello from 2')

handler = logging.StreamHandler()
handler.setLevel('INFO')

# Show me cli-circuit.1 logs
print('PRINT LOGS FROM CIRCUIT 1')
circ1 = logging.getLogger('cli-circuit.1')
circ1.addHandler(handler)
circ1.setLevel('INFO')

spam()

# Cleanup
circ1.handlers.clear()

print('PRINT LOGS FROM ALL CIRCUITS')
# Show me ANY cli-circuit logs.
all_circ = logging.getLogger('cli-circuit')
all_circ.addHandler(handler)
all_circ.setLevel('INFO')

spam()
```

Output:

```
$ python demo.py
PRINT LOGS FROM CIRCUIT 1
hello from 1
PRINT LOGS FROM ALL CIRCUITS
hello from 1
hello from 2